vhost-device-gpu/src/main.rs

```diff
@@ -158,9 +161,9 @@ mod tests {
         }
 
         // Convert each CapsetName into GpuCapset
-        for capset_name in CapsetName::value_variants().iter().cloned() {
+        for capset_name in CapsetName::value_variants().iter().copied() {
             let resulting_capset: GpuCapset = capset_name.into(); // Would panic! if the definition is incorrect
-            assert_eq!(resulting_capset.bits(), capset_name as u64)
+            assert_eq!(resulting_capset.bits(), capset_name as u64);
```

Comment:
nit: unrelated, BTW I'm fine with this change, so let's keep it, but please in the future try to avoid these changes in a patch that already touches around 3000 lines of code...

Reply:
I put it in a seperate commit, Thanks!